engine/runtime/src/main/java/org/enso/interpreter/node/callable/SequenceLiteralNode.java

```diff
@@ -35,9 +41,19 @@ public static SequenceLiteralNode build(ExpressionNode[] items) {
   @ExplodeLoop
   public Object executeGeneric(VirtualFrame frame) {
     Object[] itemValues = new Object[items.length];
+    ArrayRope<Warning> warnings = new ArrayRope<>();
     for (int i = 0; i < items.length; i++) {
       itemValues[i] = items[i].executeGeneric(frame);
+      if (itemValues[i] instanceof WithWarnings) {
+        WithWarnings withWarnings = (WithWarnings) itemValues[i];
```

Comment:
```suggestion
      if (itemValues[i] instanceof WithWarnings withWarnings) {
```

Any reason not to use the new Java features if we are already compiling with Frgaal?